### ISSUE TITLE
KeyCloak Integration for Stack Overflow - SAML assertions

### ISSUE THREAD

**Issue by NickCorcoran** [BCDevOps/developer-experience#2502]
- [ ] Investigate where SAML assertions need to live and how passed to SO
- [ ] Create test assertions and co-incident role-bindings in Stack Overflow
- [ ] Test out

Once above is complete, we'll need another ticket to do the following:
- [ ] Create default SAML assertion for new users (majority will fit this profile)
- [ ] Create Admin assertion
- [ ] Create other assertions as required
- [ ] Test out

**Comment by NickCorcoran** [comment]
This was done as part of work w/ Marco under another ticket.
